DQMOffline/Lumi/plugins/ZCounting.cc

```diff
@@ -609,8 +609,8 @@ void ZCounting::analyzeElectrons(const edm::Event& iEvent, const edm::EventSetup
   }
 
   // Good vertex requirement
-  if (nvtx == 0)
-    return;
+  //if (nvtx == 0)
+  //  return;
```

Comment:
can you please remove these two comment lines?

Reply:
The two lines were been removed.